### ISSUE TITLE
Concerning the entry "Saving Entire Document Trees" of the page "Hints":

### ISSUE THREAD

**Issue by pascallothar** [danny0838/firefox-scrapbook-maf-creator#2]
In the page [https://github.com/danny0838/firefox-scrapbook-maf-creator/wiki/Hints#saving-entire-document-trees], the entry **"Saving Entire Document Trees"** needs :-) some comments:

You say _**"Save the resulting page in Scrapbook"**_ (meaning the _"Output Tree as HTML..."_), but the saved page will only contain a single HTML page!!! So people should not delete the indexed items after having created the .maff, otherwise it would point on only a bunch of _"file not found"_ ==> big data lost!!! :-) or :-( !

So, when you say _**"You will now have a MAFF containing all of the selected pages and a convenient menu."**_, it will be true only if you knew that you had to choose _"In-Depth Capture"_!!!

But, if one of the items was captured with _"In-Depth Capture"_ and _"Depth to follow links" == 3_ , you will not be able to capture it (with your hint) with a depth higher than 2 (3 minus 1), resulting on a lost of information in comparison to the original tree.

You will also loose information because the `index.dat` inside each item will not be saved. So you will loose _comments_, creation date, etc on each item (but, the _source URL_ will still be accessible via the _green arrows_ in the _"Output Tree as HTML..."_).

I think you make a great job in maintaining and improving the legacy of Scrapbook and Scrapbook Plus and **_you have to be thanked for this_**, but I think also that your add-on desserves much better than this misleading hint :-) . And, actually, other features of your add-ons need only some little tweaks to achieve the purpose of this hint (see #3 ).

Other thing, you also need to capture with _"Scripts" checked_, otherwise the _"Output Tree as HTML..."_ index will not expand and the items will not be accessible (they are although saved). If you don't want to check _"Scripts"_, you can of course expand the index before capturing, but you will not be able to collapse parts of the index (if this one is very long).

And also, but, it is only an no important feedback, when you use the _search box_ from inside the saved _"Output Tree as HTML..."_, it will output (which is a behavior that will not astonish us):
    • Unable to load database: ./cache.rdf
    • Unable to load database: ./scrapbook.rdf
    • Error: Unable to load any database. Maybe your browser doesn't support AJAX connection?

And also, when capturing, you will probably want to not capture again the external URLs, to avoid duplicates. So you will have to _check "Filter"_, choose _"Include"_, fill the text field with _"file:///"_, perhaps also uncheck the file ending with _"search.html"_, click _"Accept"_, then click _"Start"_... and wait a long time without canceling because you thought something is going wrong!

By the way, when you capture the output of _"Output Tree as HTML..."_ (without having expanded the tree) (with _"Scripts" checked_ and _depth == 0_), the `folder_open.png` icon doesn't show up, making the indent difficult to follow. Also a different color for `folder_open.png` and `folder.png` would make the "reading" better. And similarly when you capture the output after having expanded the tree (with _"Scripts" checked_ and _depth == 0_), the `folder.png` icon doesn't show up. You will need to add them by hand by copying them from `ScrapBook/tree/folder_open.png` and `ScrapBook/tree/folder.png`
(**EDIT:** This bebaviour doesn't appear with **"Convert current Scrapbook to .maff"**).

And also, when you capture the output of _"Output Tree as HTML..."_ ("Scripts" checked or not, even with _depth == 0_) and you had in your tree an item which is the capture from a .maff open in a Firefox window, then you will find a copy of this .maff in the repertory of your capture. The normal behaviour would be to have the _green arrow_ (source) pointing to the .maff on your computer and not to duplicate it inside your capture. (See also [https://github.com/danny0838/firefox-scrapbook/issues/107]).
(**EDIT:** This bebaviour doesn't appear with **"Convert current Scrapbook to .maff"**).

**Comment by danny0838** [comment]
You are right. The page was migrated from legacy Scrapbook MAF Creator, and I forgot to remove the misleading hints. For a quick solution I removed the page. Thank you for pointing this out.
